spec/features/admin/members/admin_search_member_spec.rb

```diff
@@ -20,9 +20,9 @@
     scenario "Search with non-matching data" do
       fill_in('Email', with: current_user.email)
       click_button('Search')
-      expect(page).to have_content("No users matched that search")
+      expect(page).to have_content("No users matched your search")
```

Comment:
Because our searches aren't always a single search term. They are much more commonly going to be a selection of criteria